src/main/scala/no/ndla/draftapi/controller/DraftController.scala

```diff
@@ -628,5 +628,62 @@ trait DraftController {
       }
     }
 
+    post(
+      "/partial-publish/:article_id",
+      operation(
+        apiOperation[Article]("partialPublish")
+          summary "Partial publish selected fields"
+          description "Partial publish selected fields"
+          parameters (
+            asHeaderParam(correlationId),
+            asPathParam(articleId),
+            asQueryParam(language),
+            asQueryParam(fallback)
+        )
+          authorizations "oauth2"
+          responseMessages (response404, response500)
+      )
+    ) {
+      val userInfo = user.getUser
+      val articleId = long(this.articleId.paramName)
+      val language = paramOrDefault(this.language.paramName, Language.AllLanguages)
+      val fallback = booleanOrDefault(this.fallback.paramName, default = false)
+
+      doOrAccessDenied(userInfo.canWrite) {
+        writeService.partialPublish(articleId, language, fallback) match {
```

Comment:
Kunne vore praktisk å kunne fortelle endepunktet kva for felt som skal publiseres. Men det kan sikkert tas i ein seinare versjon.

Reply:
Foreløpig så er det jo bare det ene feltet, så tenker at vi kan vente med den logikken til vi trenger den?